lms/djangoapps/courseware/tabs.py

```diff
@@ -41,6 +41,18 @@ class CoursewareTab(EnrolledTab):
     is_default = False
     supports_preview_menu = True
 
+    def __init__(self, tab_dict):
+        def link_func(course, reverse_func):
+            if course_home_mfe_outline_tab_is_active(course.id):
+                return get_microfrontend_url(course_key=course.id, view_name=self.view_name)
+            else:
+                reverse_name_func = lambda course_test: default_course_url_name(course_test.id)
+                url_func = course_reverse_func_from_name_func(reverse_name_func)
+                return url_func(course, reverse_func)
```

Comment:
@Dillon-Dumesnil @mikix 
I made a few small changes, but a re-review would be nice on these lines for Python sanity.

Previously, because this was a property, the lambda that was returned was being executed when `.link_func` was called and evaluating to a string. Now that it's a function, calling `.link_func` executes this function and it evaluates to a lambda. My fix was to just call the lambda here...